test/parallel/test-tls-honorcipherorder.js

```diff
@@ -46,54 +43,50 @@ function test(honorCipherOrder, clientCipher, expectedCipher, cb) {
       coptions.ciphers = clientCipher;
     }
     const port = this.address().port;
-    const client = tls.connect(port, localhost, coptions, function() {
+    const savedDefaults = tls.DEFAULT_CIPHERS;
+    tls.DEFAULT_CIPHERS = defaultCiphers || savedDefaults;
+    const client = tls.connect(port, localhost, coptions, mustCall(function() {
       const cipher = client.getCipher();
       client.end();
       server.close();
-      assert.strictEqual(cipher.name, expectedCipher);
-      if (cb) cb();
-    });
-  });
+      const msg = util.format(
+        'honorCipherOrder=%j, clientCipher=%j, expect=%j, got=%j',
+        honorCipherOrder, clientCipher, expectedCipher, cipher.name);
```

Comment:
Can you indent by four spaces?  (I think the linter is supposed to complain.  Not 100% sure though.)